content/faq.md

```diff
@@ -8,9 +8,10 @@ toc = "true"
 
 # Common problems
 
-## Cannot open display
+## Errors about missing or failing to open a display
 
-Dunst was unable to connect to the X server. Make sure that X is running and that the `DISPLAY` environment variable is correctly set.
+Depending on the setup dunst was unable to connect to the X server or the wayland compositor in use. In case of X make sure that X is running and that the `DISPLAY` environment variable is correctly set. For a wayland based setup the `WAYLAND_DISPLAY` environment variable needs to be set to a valid value.  
+Distributions which use `systemd` can use `systemctl --user import-environment DISPLAY WAYLAND_DISPLAY` to get and set those environment variables with the right value.
```

Comment:
Only if you're actually starting Dunst with systemd or dbus. Could you add that detail and merge this with the next sentence below?
Another nitpick is that you only need DISPLAY for X11 setups and only WAYLAND_DISPLAY for wayland setups. If you can add that it's nice but otherwise it's fine as well.
Other than that it looks good, thank you.

Reply:
Thank you for the feedback. Good point you make in the first sentence.
And in a first draft I had it more separated in regards of importing env vars. Which kinda looked redundant. And as `DISPLAY` may be a backup on wayland if xwayland uses it, I decided to move this into one sentence.
But I will take another jab at it and see what I can come up with your points in mind.